crates/tabby-common/src/registry.rs

```diff
@@ -94,6 +138,23 @@ impl ModelRegistry {
             .join(GGML_MODEL_RELATIVE_PATH.as_str())
     }
 
+    pub fn migrate_model_path(
+        &self,
+        name: &str,
+        old_model_path: &PathBuf,
+    ) -> Result<(), std::io::Error> {
+        // legacy model always has a single file
+        let model_path = self
+            .get_model_store_dir(name)
+            .join("model-00001-of-00001.gguf");
```

Comment:
```suggestion
            .join("model-00000-of-00001.gguf");
```

shard file shall be 0 based